packages/modular-scripts/src/cli.ts

```diff
@@ -230,10 +245,9 @@ function addApp(name: string) {
 function test(args: string[]) {
   const modularRoot = getModularRoot();
 
-  let argv = mri(process.argv.slice(3))._.concat([
-    '--config',
-    path.join(__dirname, '..', 'jest-config.js'),
-  ]);
+  let argv = process.argv
+    .slice(3)
+    .concat(['--config', path.join(__dirname, '..', 'jest-config.js')]);
```

Comment:
trying to pass an additional config on top of this refuses to run jest, so this is actually fine and expected behaviour (tho we could add an error message to be nice. maybe later.)

Reply:
Let's raise an issue for this? Would be a good "first issue" for someone. 😄

Reply:
I suppose, yeah. It's not supported behaviour in create-react-app either, so we should mirror whatever that behaviour is. I'll file something tomorrow (or I may just fix it).

Reply:
I checked, and create-react-app doesn't do anything differently either, so we shouldn't too. 

But, if (and this is a big if) we do move away from create-react-app, we can and should rethink the experience. 

Marking this conversation as resolved.